lib/render/VolumeRenderer.cpp

```diff
@@ -180,6 +180,15 @@ int VolumeRenderer::_paintGL(bool fast)
     return ret;
 }
 
+std::string VolumeRenderer::_getColorbarVariableName() const
+{
+    VolumeParams *vp = (VolumeParams *)GetActiveParams();
```

Comment:
@shaomeng you've asked for a change to this PR, but you've also approved the PR. It will help me if you don't approve the PR until after you have "resolved" all conversations. Thanks!

Reply:
Gotcha. Un-approve it now!

Reply:
Done